.github/workflows/bump-and-publish-versions-prod.yml

```diff
@@ -5,6 +5,9 @@ on:
     - cron: '5 * * * *'
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.head_ref }}
```

Comment:
Bump versions can put new commits on main, which subsequent invocations would run under. Do we want this to use a static string instead?

Reply:
@cam72cam 

I chose the dynamic value since I noticed the `workflow_dispatch`.
If there are no plans to do runs from branches other than the main, I can hardcode.

Reply:
I just took a closer look athttps://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value.

We may want to run it on branches as validation at some point, but for now a global lock should suffice.  We should probably add a brief comment about this conversation in the file.